!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  blendFront: 0.1
  blendBack: 0.0
  correction1: 0.0
  correction2: 0.0
  correction3: 0.0
  correction4: 0.0
  croppedHeight: 617.0
  croppedWidth: 633.0
  fovIn: 195.0
  fovOut: 90.0
  gridLines: false
  inputProjection: 1
  outputProjection: 0
  pitch: 0.0
  roll: 90.0
  x: 1.0
  xCenter: 958.5
  y: 1.0
  yCenter: 321.5
  yaw: 0.0
  z: 1.0
state: []
